app.js

```diff
@@ -0,0 +1,37 @@
+// Import function from other file 
+import { compareNumbers } from './function.js'; 
+
+// reference DOM elements 
+const endPhrase = document.getElementById('end-phrase');
+const guessInput = document.getElementById('guess-number');
+const submitButton = document.getElementById('button');
+const countRemaining = document.getElementById('remaining');
+
+// reference compareNumbers function from imported file function.js
+let rightAnswer = Math.ceil(Math.random() * 20); 
+let numberOfGuessesRemaining = 4; 
+
+function compareNumbers (guess, rightAnswer) {
```

Comment:
if you import it above, you do not need to re-paste the code in the document here.  you would pass these parameters in when you invoke the function on line 28.